### DISCUSSION TITLE
[RELEASE] Final Stand

### DISCUSSION THREAD

**Discussion by McLyntree** [nzp-team/nzportable#771] [Map Releases]
# Final Stand
> A marine recon unit seeks refuge at the ruined Hohenlychen Sanatorium following the outbreak in Wittenau, mounting a fleeting defense against waves of the relentless undead...

My first mapping endeavor is available to play (excluding some final detailing work).
Based on the horrors of _Verrückt_, this one-room, four barricade map beckons you to hold out as long as you can.

_(NOTE: This map has yet to be tested on platforms other than PC, please send any issues my way if they pop up)_

![Screenshot1](https://github.com/nzp-team/nzportable/assets/131576074/d1fc3768-e4eb-4245-9d31-eb8b959198ba)
![Screenshot2](https://github.com/nzp-team/nzportable/assets/131576074/5315346c-3395-483d-874b-a1b98d7095da)
![Screenshot3](https://github.com/nzp-team/nzportable/assets/131576074/44ca3bec-075a-4bef-a430-304da9eb7342)
![Screenshot4](https://github.com/nzp-team/nzportable/assets/131576074/d79627b8-aeed-4f06-bf9b-429a1232b7f4)

# Download
_A few extras, such as the `.map` file for anyone to view or edit, is also included in the download below:_
[Download Link](https://archive.org/download/final-stand_202506/FinalStand.zip)

**Comment by MotoLegacy** [comment]
This map nails the uneasy setting it was inspired by. I didn't think NZ:P could really  illustrate much in terms of horror with our current art style. Very good. :)

**Reply by McLyntree** [reply]
Thank you, I remember when I first launched the map with the tinted lighting it immediately felt more eerie than I thought it would - the claustrophobic layout also probably helped with that too.

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org

**Comment by Jeyphr** [comment]
Check out my video review here! https://youtu.be/5pTNkID_VDQ
( When it comes out )

**Reply by Jeyphr** [reply]
It's out now!

**Comment by McLyntree** [comment]
**6/16/25 Update**
 - Download link was moved over to archive.org instead of github.com.

**Comment by scootsterling** [comment]
Played for a few minutes on my PS Vita, seems to work fine, just had to rename "map" folder to "maps"

**Comment by nonMat** [comment]
Pretty cool map!!!
Kind of like a Verruckt lite!
